app/measurement/views.py

```diff
@@ -158,41 +158,50 @@ def list(self, request, *args, **kwargs):
         return super().list(self, request, *args, **kwargs)
 
 
-class ThresholdViewSet(BaseMeasurementViewSet):
+class ThresholdViewSet(MeasurementBaseViewSet):
     serializer_class = serializers.ThresholdSerializer
     filter_class = ThresholdFilter
 
     def get_queryset(self):
         return Threshold.objects.all()
 
 
-class MonitorViewSet(BaseMonitorViewSet):
+class MonitorViewSet(MonitorBaseViewSet):
     serializer_class = serializers.MonitorSerializer
     filter_class = MonitorFilter
 
     def get_queryset(self):
-        return Monitor.objects.all()
+        queryset = Monitor.objects.all()
+        if self.request.user.is_staff:
+            return queryset
```

Comment:
Does this mean a staff user would get all Monitors, or just all Monitors within an organization? Same goes for other classes with the same is_staff check.

Reply:
Yes, staff is the user model attr for admin

Reply:
It's an unfortunate name, since 'staff' implies a user of an organization, but it's the default name in the django user model.  It's the django way of differentiating between a website user and a user who can access the admin section.

Reply:
Ok, sounds good